docs/examples/hello_numpy.rst

```diff
@@ -118,23 +118,22 @@ Here, in ``executors``, the Trainer implementation ``NPTrainer`` is configured f
 Federated Numpy with Scatter and Gather Workflow!
 -------------------------------------------------
 
-Now you can use admin commands to upload, deploy, and start this example app. To do this on a proof of concept local
+Now you can use admin command prompt to submit and start this example app. To do this on a proof of concept local
 FL system, follow the sections :ref:`setting_up_poc` and :ref:`starting_poc` if you have not already.
 
 Running the FL System
 ^^^^^^^^^^^^^^^^^^^^^
 
-With the admin client command prompt successfully connected and logged in, enter the commands below in order.  Pay close
-attention to what happens in each of four terminals.  You can see how the admin controls the server and clients with
-each command.
+With the admin client command prompt successfully connected and logged in, enter the command below.  Pay close
+attention to what happens in each of four terminals. You can see how the admin submits the job to the server and how
+the JobRunner on the server automatically picks up the job to deploy and start the run.
```

Comment:
I notice these sections has the same content.
Is it possible to move these common sections into separate file and include them here?

something like: `.. include:: base.rst`